spoon-runner/src/main/java/com/squareup/spoon/SpoonRunner.java

```diff
@@ -215,6 +222,7 @@ private SpoonDeviceRunner getTestRunner(String serial, SpoonInstrumentationInfo
     private IRemoteAndroidTestRunner.TestSize testSize;
     private int adbTimeout;
     private boolean failIfNoDeviceConnected;
+    private List<ITestRunListener> testRunListeners;
```

Comment:
You can just initialize this to `new ArrayList<>` and then you don't need the null checks all over the place.